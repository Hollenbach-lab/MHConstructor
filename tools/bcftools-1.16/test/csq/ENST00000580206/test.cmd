{bin}/bcftools csq -f ENST00000580206.fa -g ENST00000580206.gff compound.ins-snv.vcf  | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -f ENST00000580206.fa -g short.gff compound.ins-snv.vcf  | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -f ENST00000580206.fa -g ENST00000580206.gff single.ins-snv.vcf  | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -f ENST00000580206.fa -g short.gff single.ins-snv.vcf  | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -f ENST00000580206.fa -g ENST00000580206.gff compound.del-snv.vcf  | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -f ENST00000580206.fa -g short.gff compound.del-snv.vcf  | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -f ENST00000580206.fa -g short.gff single.del-snv.vcf  | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -f ENST00000580206.fa -g ENST00000580206.gff compound.del-ins.vcf  | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -f ENST00000580206.fa -g short.gff compound.del-ins.vcf  | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -f ENST00000580206.fa -g short.gff single.del-ins.vcf  | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
