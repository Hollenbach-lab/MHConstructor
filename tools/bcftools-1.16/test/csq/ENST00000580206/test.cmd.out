35750	C	CAAAAGA	inframe_insertion|ANKRD30B|ENST00000358984|protein_coding|+|353P>353QKK|35750C>CAAAAGA+35751C>A,inframe_insertion|ANKRD30B|ENST00000580206|NMD|+|353P>353QKK|35750C>CAAAAGA+35751C>A,inframe_insertion|ANKRD30B|ENST00000665241|protein_coding|+|353P>353QKK|35750C>CAAAAGA+35751C>A
35751	C	A	@35750

35750	C	CAAAAGA	inframe_insertion|ANKRD30B|ENST00000358984|protein_coding|+|353P>353QKK|35750C>CAAAAGA+35751C>A
35751	C	A	@35750

35750	CC	CAAAAGAA	inframe_insertion|ANKRD30B|ENST00000358984|protein_coding|+|353P>353QKK|35750CC>CAAAAGAA,inframe_insertion|ANKRD30B|ENST00000580206|NMD|+|353P>353QKK|35750CC>CAAAAGAA,inframe_insertion|ANKRD30B|ENST00000665241|protein_coding|+|353P>353QKK|35750CC>CAAAAGAA

35750	CC	CAAAAGAA	inframe_insertion|ANKRD30B|ENST00000358984|protein_coding|+|353P>353QKK|35750CC>CAAAAGAA

35750	CCAG	C	inframe_deletion|ANKRD30B|ENST00000358984|protein_coding|+|353PA>353P|35750CCAG>C+35755A>T,inframe_deletion|ANKRD30B|ENST00000580206|NMD|+|353PA>353P|35750CCAG>C+35755A>T,inframe_deletion|ANKRD30B|ENST00000665241|protein_coding|+|353PA>353P|35750CCAG>C+35755A>T
35755	A	T	@35750
35756	A	C	missense|ANKRD30B|ENST00000358984|protein_coding|+|355K>354Q|35756A>C,missense|ANKRD30B|ENST00000580206|NMD|+|355K>354Q|35756A>C,missense|ANKRD30B|ENST00000665241|protein_coding|+|355K>354Q|35756A>C

35750	CCAG	C	inframe_deletion|ANKRD30B|ENST00000358984|protein_coding|+|353PA>353P|35750CCAG>C+35755A>T
35755	A	T	@35750
35756	A	C	missense|ANKRD30B|ENST00000358984|protein_coding|+|355K>354Q|35756A>C

35750	CCAGCA	CCT	inframe_deletion|ANKRD30B|ENST00000358984|protein_coding|+|353PA>353P|35750CCAGCA>CCT
35756	A	C	missense|ANKRD30B|ENST00000358984|protein_coding|+|355K>354Q|35756A>C

35750	CCAG	C	missense&inframe_altering|ANKRD30B|ENST00000358984|protein_coding|+|353PA>353PP|35750CCAG>C+35755A>ACCT,missense&inframe_altering|ANKRD30B|ENST00000580206|NMD|+|353PA>353PP|35750CCAG>C+35755A>ACCT,missense&inframe_altering|ANKRD30B|ENST00000665241|protein_coding|+|353PA>353PP|35750CCAG>C+35755A>ACCT
35755	A	ACCT	@35750

35750	CCAG	C	missense&inframe_altering|ANKRD30B|ENST00000358984|protein_coding|+|353PA>353PP|35750CCAG>C+35755A>ACCT
35755	A	ACCT	@35750

35750	CCAGCA	CCACCT	missense|ANKRD30B|ENST00000358984|protein_coding|+|354A>354P|35750CCAGCA>CCACCT
